servicenow/resources/resource_scripted_rest_api.go

```diff
@@ -0,0 +1,138 @@
+package resources
+
+import (
+	"github.com/coveooss/terraform-provider-servicenow/servicenow/client"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+const scriptedRestApiActive = "active"
+const scriptedRestApiConsumes = "consumes"
+const scriptedRestApiConsumes_customized = "consumes_customized"
+const scriptedRestApiEnforce_acl = "enforce_acl"
+const scriptedRestApiName = "name"
+const scriptedRestApiProduces = "produces"
+const scriptedRestApiProduces_customized = "produces_customized"
+
+// ResourceScriptedRestApi manages a System Property in ServiceNow.
+func ResourceScriptedRestApi() *schema.Resource {
+	return &schema.Resource{
+		Create: createResourceScriptedRestApi,
+		Read:   readResourceScriptedRestApi,
+		Update: updateResourceScriptedRestApi,
+		Delete: deleteResourceScriptedRestApi,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			scriptedRestApiActive: {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Activates the API. Inactive APIs cannot serve requests.",
+			},
+			scriptedRestApiConsumes: {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Default supported request formats.",
+			},
+			scriptedRestApiConsumes_customized: {
```

Comment:
The "Customized" booleans can be inferred, if nothing is entered in the Consumes, then it can be set to false? Or else always send true and put a default value in consume on the tf side